src/graphql/query.rs

```diff
@@ -21,6 +21,13 @@ graphql_object!(QueryRoot: RequestContext as "Query" |&self| {
         Ok(crate::resolvers::project::all(&database)?)
     }
 
+    field media(
+        &executor
+    ) -> FieldResult<Vec<Medium>> {
+        let database = &executor.context().conn;
+        Ok(crate::resolvers::medium::all(&database)?)
```

Comment:
Could you refactor this to use the `database_context` method of the `RequestContext` struct? i.e.

```rust
        let database_context = &executor.context().database_context(ActionType::Read, ActionModifier::All);
        Ok(crate::resolvers::medium::all(&database_context)?)
```

Reply:
We'll wait for your other PR to get merged, then I'll rebase and fix accordingly